dist/js/jsflap.js

```diff
@@ -1557,6 +1597,216 @@ var jsflap;
+                var offsetPoint = new jsflap.Point.IMPoint(minX, minY);
+                JSONData += '{board : {\n'
+
+                var hasPassed = false;
+                var numNodes = 0;
+                var nodeData = '';
+                this.visualizations.nodes.forEach(function (node) {
+                    numNodes += 1;
+                    var pos = node.position.getMPoint().subtract(offsetPoint).round();
+                    if (hasPassed) {
+                        nodeData += ', \n';
+                    } else {
+                        nodeData += '\tNodeL : [\n';
+                    }
+                    nodeData += '\t\t{\n'
+                    nodeData += '\t\tName : ' + node.model.label + ', \n'
+                    nodeData += '\t\tXCoor : ' + pos.x + ', \n';
+                    nodeData += '\t\tYCoor : ' + pos.y + ', \n';
+                    if (node.model.final) {
+                        nodeData += '\t\tFinal : true, \n';
+                    } else {
+                        nodeData += '\t\tFinal : false, \n';
+                    }
+                    if (node.model.initial) {
+                        nodeData += '\t\tInitial : true, \n';
+                    } else {
+                        nodeData += '\t\tInitial : false, \n';
+                    }
+                    nodeData += '\t\t}';
+                    hasPassed = true;
+                }); 
+                if (hasPassed) {
+                    nodeData += '\n\t];\n';
+                }
+                JSONData += '\tnumNodes : ' + numNodes.toString() + ';\n';
+                
+
+                
+                hasPassed = false;
+                var numEdges = 0;
+                var edgeData = '';
+                this.visualizations.edges.forEach(function (edge) {
+                    numEdges += 1
+                    if (hasPassed) {
+                        edgeData += ', \n'
+                    } else {
+                        edgeData += '\tEdgeL : [\n';
+                    }
+                    edgeData += '\t\t{\n'
+                    edgeData += '\t\tStart Node : ' + edge.fromModel.label + ', \n';
+                    edgeData += '\t\tEnd Node : ' + edge.toModel.label + ', \n';
+                    edge.models.items.forEach(function (edgeModel) {
+                        edgeData += '\t\tReadValue : ' + edgeModel.transition.toString() + ', \n';
+                    });
+                    edgeData += '\t\t}';
+                    hasPassed = true;
+                });
+                if (hasPassed) {
+                    edgeData += '\n\t];\n';
+                }
+                JSONData += '\tnumEdges : ' + numEdges.toString() + ';\n';
+                JSONData += nodeData;
+                JSONData += edgeData;
+                JSONData += '}\n';
+                return JSONData;
+            };
+            Board.prototype.toJFLAP = function() {
```

Comment:
Ok, you also implimented this below in the source file, which you should move all this code there. Again, this all gets overwritten on build.